Flaky reactive test

```
[info] StreamToPublisherTest:
[info] - optional_spec104_mustSignalOnErrorWhenFails *** FAILED ***
[info]   java.lang.RuntimeException: Publisher threw exception (Error-state Publisher scalaz.zio.interop.reactiveStreams.Adapters$$anonfun$streamToPublisher$1$$anonfun$1@22cee1a1 did not call `onError` on new Subscriber within 500 ms) instead of signalling error via onError!
[info]   at org.reactivestreams.tck.PublisherVerification.optional_spec104_mustSignalOnErrorWhenFails(PublisherVerification.java:391)
[info]   at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[info]   at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[info]   at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[info]   at java.lang.reflect.Method.invoke(Method.java:498)
[info]   at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
[info]   at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
[info]   at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
[info]   at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
[info]   at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
[info]   ...
[info]   Cause: java.lang.AssertionError: Error-state Publisher scalaz.zio.interop.reactiveStreams.Adapters$$anonfun$streamToPublisher$1$$anonfun$1@22cee1a1 did not call `onError` on new Subscriber within 500 ms
[info]   at org.testng.Assert.fail(Assert.java:89)
[info]   at org.reactivestreams.tck.TestEnvironment.flop(TestEnvironment.java:169)
[info]   at org.reactivestreams.tck.TestEnvironment$Latch.expectClose(TestEnvironment.java:852)
[info]   at org.reactivestreams.tck.TestEnvironment$Latch.expectClose(TestEnvironment.java:846)
[info]   at org.reactivestreams.tck.PublisherVerification$6.run(PublisherVerification.java:381)
[info]   at org.reactivestreams.tck.PublisherVerification.potentiallyPendingTest(PublisherVerification.java:1191)
[info]   at org.reactivestreams.tck.PublisherVerification.whenHasErrorPublisherTest(PublisherVerification.java:1182)
[info]   at org.reactivestreams.tck.PublisherVerification.optional_spec104_mustSignalOnErrorWhenFails(PublisherVerification.java:361)
[info]   at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[info]   at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[info]   ...
```

--- a/interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/SinkToSubscriberTest.scala
+++ b/interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/SinkToSubscriberTest.scala
@@ -12,7 +12,7 @@ class SinkToSubscriberTest(env: TestEnvironment)
     with DefaultRuntime {
 
   def this() {
-    this(new TestEnvironment(500))
+    this(new TestEnvironment(1000, 500))
   }
 
   override def createSubscriber(): Subscriber[Int] =

--- a/interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/StreamToPublisherTest.scala
+++ b/interop-reactiveStreams/jvm/src/test/scala/scalaz/zio/interop/reactiveStreams/StreamToPublisherTest.scala
@@ -12,7 +12,7 @@ class StreamToPublisherTest(env: TestEnvironment, publisherShutdownTimeout: Long
     with DefaultRuntime {
 
   def this() {
-    this(new TestEnvironment(500), 1000)
+    this(new TestEnvironment(1000, 500), 1000)
   }
 
   def createPublisher(elements: Long): Publisher[Int] =

